examples/CIB/ex1/main.cpp

```diff
@@ -399,6 +495,83 @@ main(int argc, char* argv[])
             pout << "+++++++++++++++++++++++++++++++++++++++++++++++++++\n";
             pout << "\n";
 
+            // Fill ghost cells of pressure and velocity to compute hydrodynamic forces.
```

Comment:
The next chunk of code should be done inside the force evaluator object itself to reduce boilerplate code.